cmd/av/pr_create.go

```diff
@@ -85,12 +88,22 @@ Examples:
 				Draft:      draft,
 				Edit:       prCreateFlags.Edit,
 			},
-		); err != nil {
+		)
+		if err != nil {
 			return err
 		}
 		if err := tx.Commit(); err != nil {
 			return err
 		}
+
+		// Do this after creating the PR and committing the transaction so that
+		// our local database is up-to-date even if this fails.
+		if len(prCreateFlags.Reviewers) > 0 {
```

Comment:
Doesn't GH allow passing reviewers as part of creating the PR? I wonder if it does, would the behavior be different to do this in two separate steps. Would GH assign some default reviewers still

Reply:
> Doesn't GH allow passing reviewers as part of creating the PR?

The create pull request API (mutation `createPullRequest`) does not have a way to assign reviewers (nor does the `updatePullRequest` mutation).